src/histolab/data/__init__.py

```diff
@@ -0,0 +1,264 @@
+    # our data. A ConnectionError is raised if no internet connection is
+    # available.
+    try:
+        resolved_path = image_fetcher.fetch(data_filename)
+    except ConnectionError as err:
+        # If we decide in the future to suppress the underlying 'requests'
+        # error, change this to `raise ... from None`. See PEP 3134.
+        raise ConnectionError(
+            "Tried to download a histolab dataset, but no internet "
+            "connection is available. To avoid this message in the "
+            "future, try `histolab.data.download_all()` when you are "
+            "connected to the internet."
+        ) from err
+    return resolved_path
+
+
+def _init_pooch():
+    os.makedirs(data_dir, exist_ok=True)
+    # data_base_dir = os.path.join(data_dir, "..")
+    # Fetch all legacy data so that it is available by default
+    for filename in legacy_registry:
+        _fetch(filename)
+
+
+if has_pooch:
+    _init_pooch()
+
+
+def _load_svs(f):
+    """Load an image file located in the data directory."""
+    try:
+        svs = openslide.open_slide(_fetch(f))
+    except openslide.OpenSlideError:
+        raise openslide.OpenSlideError(
+            "Your wsi has something broken inside, a doctor is needed"
+        )
+    return svs
+
+
+def cmu_small_region():
+    """Carnegie Mellon University MRXS sample tissue
+
+    Licensed under a CC0 1.0 Universal (CC0 1.0) Public Domain Dedication.
+
+    Returns
+    -------
+    cmu_mrxs_tissue : OpenSlide object
+        Sample CMU tissue Whole-Slide-Image
+    """
+    return _load_svs("data/cmu_small_region.svs")
+
+
+def aorta_tissue():  # pragma: no cover
+    """Aorta tissue, brightfield, JPEG 2000, YCbCr
+
+    This image is avaliable here
+    http://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/
+
+    Free to use and distribute, with or without modification
+
+    Returns
+    -------
+    aorta_tissue : OpenSlide object
+        Aorta tissue Whole-Slide-Image
+    """
+    return _load_svs("aperio/JP2K-33003-1.svs")
+
+
+def heart_tissue():  # pragma: no cover
+    """Heart tissue, brightfield, JPEG 2000, YCbCr
+
+    This image is avaliable here
+    http://openslide.cs.cmu.edu/download/openslide-testdata/Aperio/
+
+    Free to use and distribute, with or without modification
+
+    Returns
+    -------
+    heart_tissue : OpenSlide object
+        Heart tissue Whole-Slide-Image
+    """
+    return _load_svs("aperio/JP2K-33003-2.svs")
+
+
+def breast_tissue():  # pragma: no cover
+    """Breast tissue, TCGA
+
+    This image is avaliable here
+    https://portal.gdc.cancer.gov/files/9c960533-2e58-4e54-97b2-8454dfb4b8c8
+    or through the API
+    https://api.gdc.cancer.gov/data/9c960533-2e58-4e54-97b2-8454dfb4b8c8
+
+    Access: open
+
+    Returns
+    -------
+    breast_tissue : OpenSlide object
+        Breast tissue Whole-Slide-Image
+    """
+    return _load_svs("tcga/breast/9c960533-2e58-4e54-97b2-8454dfb4b8c8")
```

Comment:
no extension here?

Reply:
npe this is the trick you sent me for tcga ... so no extension here to be compliant with the API get request

Reply:
well... so here the UUID becomes the filename? :( 
the point is that for TCGA the filename has a specific meaning (see https://docs.gdc.cancer.gov/Encyclopedia/pages/TCGA_Barcode/)